src/index.test.ts

```diff
@@ -189,7 +190,9 @@ test('Own peerDependencies include those of eslint-config-standard', async (t) =
       // https://github.com/microsoft/TypeScript/pull/12253
       const name = _name as keyof typeof standardPkg.peerDependencies
       const ourRange = ourPeerDeps[name]
-      t.is(ourRange, standardRange)
+      const ourMinimum = ourRange.split('>=')[1]
+      const standardMinimum = standardRange.split('^')[1]
```

Comment:
A naming concern here.
The term minimum can apply to `>=` but not to `^`, because `^` has a maximum, as well. So it's not just a minimum, but a range.

Reply:
Updated as:
```
test('Own peerDependencies include those of eslint-config-standard', async (t) => {
  const { ourPeerDeps } = await getOurDeps()
  Object
    .entries(standardPkg.peerDependencies)
    .forEach(([_name, standardDep]) => {
      // https://github.com/microsoft/TypeScript/pull/12253
      const name = _name as keyof typeof standardPkg.peerDependencies
      const ourDep = ourPeerDeps[name]
      const ourRange = ourDep.split('>=')[1]
      const standardRange = standardDep.split('^')[1]
      t.is(ourRange, standardRange)
    })
})
```